src/layouts/BasicLayout.ts

```diff
@@ -1,17 +1,26 @@
 import {ILayout} from "../ILayout";
 import {LogEntry} from "../LogEntry";
-import {logLevelToString, LogLevel} from "../LogLevel";
+import {logLevelToString} from "../LogLevel";
+import {stringify} from "../Utils";
 
 /**
  * Simple layout, that formats logs as
  * "{time} {level} [{tag}] - {message}"
  */
 export default class BasicLayout implements ILayout {
-    format(entry:LogEntry):string {
-        return this.formatDate(entry.time) + ' ' + logLevelToString(entry.level) + ' [' + entry.tag + '] - ' + entry.message;
+    format(entry:LogEntry, include_data: boolean):string {
```

Comment:
The idea was to keep API simple and mimic the log4j and others. I thing, passing some flags (especially when they lead us to repeating code in `formatData`) is not a good idea.